java/core/src/test/org/apache/orc/TestInMemoryKeystore.java

```diff
@@ -51,9 +51,12 @@ public void init() throws IOException {
     Random random = new Random(2);
     memoryKeystore =
         new InMemoryKeystore(random)
-            .addKey("key128", EncryptionAlgorithm.AES_CTR_128, "123".getBytes(StandardCharsets.UTF_8))
-            .addKey("key256", EncryptionAlgorithm.AES_CTR_256, "secret123".getBytes(StandardCharsets.UTF_8))
-            .addKey("key256short", EncryptionAlgorithm.AES_CTR_256, "5".getBytes(StandardCharsets.UTF_8));
+            .addKey("key128", EncryptionAlgorithm.AES_CTR_128,
+                    "123".getBytes(StandardCharsets.UTF_8))
+            .addKey("key256", EncryptionAlgorithm.AES_CTR_256,
+                    "secret123".getBytes(StandardCharsets.UTF_8))
+            .addKey("key256short", EncryptionAlgorithm.AES_CTR_256,
+                    "5".getBytes(StandardCharsets.UTF_8));
```

Comment:
There are multiple ways to reduce the length of lines. We can import `java.nio.charset.StandardCharsets.UTF_8` simply like the following.

https://github.com/apache/orc/blob/522d759f13c6e52c7c9f862ecbfe493aad7ad2a2/java/core/src/test/org/apache/orc/impl/filter/leaf/ATestLeafFilter.java#L34

Moreover, for the rest of this file, can we simply import `AES_CTR_128` and `AES_CTR_256`, too?
```java
import static org.apache.orc.EncryptionAlgorithm.AES_CTR_128;
import static org.apache.orc.EncryptionAlgorithm.AES_CTR_256;
```